.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Question
+about: What do you want to know?
+title: "[QUESTION]<Enter comprehensive title>"
+labels: question
+assignees: ''
+
+---
+
```

Comment:
* [ ] I searched in open and closed issues and couldn't find anything that matches.